demos/asciidoc/didact-demo.didact.adoc

```diff
@@ -48,7 +48,13 @@ link:didact://?commandId=vscode.didact.scaffoldProject&srcFilePath=demo/projectw
 
 * link:didact://?commandId=vscode.didact.startTerminalWithName&completion=Opened%20a%20new%20terminal.[You can open integrated terminals.]
 
-* link:didact://?commandId=vscode.didact.sendNamedTerminalAString&text=newTerm$$echo%20Hello%20Didact!&completion=Opened%20a%20new%20terminal.[You can can even open a terminal, name it, and send it some text.]
+* link:didact://?commandId=vscode.didact.sendNamedTerminalAString&text=newTerm$$echo%20Hello%20Didact!&completion=Opened%20a%20new%20terminal.[You can even open a terminal, name it, and send it some text.]
+
+* link:didact://?commandId=vscode.didact.openNamedOutputChannel&text=newOutputChannel&completion=Opened%20a%20new%20output%20channel.[You can also open an output channel and name it.]
```

Comment:
this description is misleading. It is not showing an existing output channel which is the main use case that users will need. it is creating a new output channel, or showing an output channel that was created through this command.

Reply:
given that it has been already merged, i created a bug ticket https://issues.redhat.com/browse/FUSETOOLS2-714

Reply:
It is showing existing output channels. The only limitation is that you can only show those you created yourself. Btw. that PR has been merged and it is quite pointless to now start commenting on it.

Reply:
it was merged by dismissing a "requested changes" review. No chance to review it.
I created an issue to work on it to fix it

Reply:
Its hard to track every requested change because sometimes those are just too many :) Thanks for creating the issue.